rules.d/90-vconsole.rules.in

```diff
@@ -9,4 +9,4 @@
 
 # Each vtcon keeps its own state of fonts.
 #
-ACTION=="add", SUBSYSTEM=="vtconsole", KERNEL=="vtcon*", RUN+="{{ROOTLIBEXECDIR}}/systemd-vconsole-setup"
+ACTION=="add", SUBSYSTEM=="vtconsole", KERNEL=="vtcon*", RUN+="{{SYSTEMCTL_BINARY_PATH}} restart systemd-vconsole-setup.service"
```

Comment:
We need `restart`: we want the unit to run every time we discover the console. With `try-restart`, it'd never run, because nothing would start it in the first place.

Reply:
It's not useful to start it at boot. We want to start it after each of the `vtcon*` devices is reported by the kernel. (At least this is what the comments and git commit messages say.)